src/coreclr/vm/eeconfig.cpp

```diff
@@ -477,7 +481,7 @@ HRESULT EEConfig::sync()
 
     pReadyToRunExcludeList = NULL;
 #if defined(FEATURE_READYTORUN)
-    if (ReadyToRunInfo::IsReadyToRunEnabled())
+    if (g_pConfig->ReadyToRun())
```

Comment:
I do not this works - this will use `fReadyToRun` flag, but this flag is only initialized later in this method

Reply:
Makes sense, I moved the initialization above this line, please let me know if that's fine